meson.build

```diff
@@ -50,10 +50,15 @@ config_h.set_quoted('LIBRATBAG_DATA_DIR', libratbag_data_dir)
 pkgconfig = import('pkgconfig')
 dep_udev = dependency('libudev')
 dep_libevdev = dependency('libevdev')
-dep_libsystemd = dependency('libsystemd', version : '>=227')
 dep_glib = dependency('glib-2.0')
 dep_lm = cc.find_library('m')
 
+if get_option('elogind')
+	logind_provider_dep = dependency('libelogind', version : '>=227')
+else
+	logind_provider_dep = dependency('libsystemd', version : '>=227')
```

Comment:
oh, and this can just be `logind_dep` imo, that's clear enough too. Not that it matters that much.